reactor-netty-core/src/test/java/reactor/netty/tcp/TcpClientTests.java

```diff
@@ -1372,4 +1372,33 @@ private void doTestSharedNameResolver(TcpClient client, boolean sharedClient) th
 			    .block();
 		}
 	}
+
+	/* https://github.com/reactor/reactor-netty/issues/1765 */
+	@Test
+	void noSystemProxySettings() {
+		DisposableServer disposableServer =
+				TcpServer.create()
+				         .port(0)
+				         .handle((req, res) -> res.sendString(Mono.just("noSystemProxySettings")))
+				         .bindNow();
+
+		AtomicReference<AddressResolverGroup<?>> resolver = new AtomicReference<>();
+		Connection conn = null;
+		try {
+			conn = TcpClient.create()
+			                .host("localhost")
+			                .port(disposableServer.port())
+			                .proxyWithSystemProperties()
```

Comment:
should there be an assertion that the current test environment doesn't have any proxy-related system properties ? Alternatively we can test `proxyWithSystemProperties(Properties)` instead of `proxyWithSystemProperties()`.